jellydoc-maven-plugin/src/main/java/org/jvnet/maven/jellydoc/TagXMLDoclet.java

```diff
@@ -61,14 +73,16 @@
 
 // #### somehow we need to handle taglib inheritence...
 
-public class TagXMLDoclet extends Doclet {
+public class TagXMLDoclet implements Doclet {
+
+    private DocTrees docTrees;
+    private Reporter reporter;
 
     private String targetFileName = null;
     private String encodingFormat;
 
-    public TagXMLDoclet (RootDoc root) throws Exception
+    private void main(DocletEnvironment root) throws Exception
```

Comment:
Good luck trying to understand the changes to this file. The only [migration guide](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.javadoc/jdk/javadoc/doclet/package-summary.html#migration) that exists has almost zero information other than the package changes. I had to figure out all of the semantic changes by trial and error.